Makefile

```diff
@@ -25,13 +47,14 @@ STRIP_EXEC= strip -x
 DEP_TO_BIN=
 DEP_TO_LIB=
 EMU_TO_BIN= jsonrpc-remote-cartesi-machine remote-cartesi-machine remote-cartesi-machine-proxy merkle-tree-hash
-EMU_TO_LIB= $(LIBCARTESI_$(UNAME)) $(LIBCARTESI_PROTOBUF_$(UNAME)) $(LIBCARTESI_GRPC_$(UNAME))
-EMU_LUA_TO_BIN= cartesi-machine-tests.lua cartesi-machine.lua cartesi-machine-stored-hash.lua rollup-memory-range.lua uarch-riscv-tests.lua
-EMU_TO_LUA_PATH= cartesi/util.lua cartesi/proof.lua
+EMU_TO_LIB= $(LIBCARTESI_SO_$(UNAME)) $(LIBCARTESI_SO_PROTOBUF_$(UNAME)) $(LIBCARTESI_SO_GRPC_$(UNAME))
+EMU_LUA_TEST_TO_BIN= cartesi-machine.lua cartesi-machine-tests.lua uarch-riscv-tests.lua cartesi-machine-stored-hash.lua rollup-memory-range.lua
```

Comment:
The files `cartesi-machine-tests.lua` and `uarch-riscv-tests.lua` are always being installed, even though I did not do `make install-tests`.